sdk/identity/identity/src/credentials/azureCliCredential.ts

```diff
@@ -159,7 +159,7 @@ export class AzureCliCredential implements TokenCredential {
                 (err as Error).message || "Unknown error while trying to retrieve the access token"
               );
         logger.getToken.info(formatError(scopes, error));
-        throw error;
+        throw new CredentialUnavailableError(error);
```

Comment:
change the `new Error` above to `new CredentialUnavailableError` instead. Otherwise, you could potentially wrap a CredentialUnavailableError again?

Reply:
Fixed the double wrapping of Credential Unavailable Error happening.